src/components/AssetsComponent.jsx

```diff
@@ -108,7 +119,7 @@ export class AssetsComponent extends Component {
           </div>
           <AssetsTableContent
             {...this.props}
-            activePage={this.state.activePage}
+            activePage={this.getCurrentPage}

```

Comment:
Calling this `activePage` and passing it a function is misleading. Maybe rename the prop to `onGetActivePage`. However, I don't think we'll need this once you set the `activePage` in the store